### ISSUE TITLE
Panel: no drop zone for views after the last panel

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#232590]
Steps to Reproduce:

1.  drag a view container such as search to the panel
2. try to drop it as last panel

=> this is hard, you have to drop it directly after the last panel, you cannot use any of the empty space to drop

![Image](https://github.com/user-attachments/assets/e6c2b13a-f290-4417-bc2f-e80019ed8d24)

This works fine in primary sidebar and secondary sidebar.

**Comment by benibenj** [comment]
The problem is that in the panel flex grow is used in a different way than in the side bar. In the Side bar the dom element of the activity bar can grow and span the entire empty space. For the panel it's the actions which span the entire empty space. I don't know why this was done for the panel, I assume because of some specific actions such as the filter input box for the debug console.

**Comment by bpasero** [comment]
Yeah and I am not suggesting to change that, but maybe the drop zone could somehow cover that area too.
